ext/src/main/java/dev/brahmkshatriya/echo/extension/YoutubeExtension.kt

```
@@ -421,144 +429,74 @@ class YoutubeExtension : ExtensionClient, HomeFeedClient, TrackClient, SearchFee
-        components.likeManager.setLiked(track, isLiked)
+    override suspend fun searchLyrics(query: String): Feed<Lyrics> {
+        return Feed(emptyList()) { _ -> PagedData.Single { emptyList<Lyrics>() }.toFeedData() }
     }
 
-    override suspend fun listEditablePlaylists(track: Track?): List<Pair<Playlist, Boolean>> {
-        return components.playlistManager.getEditablePlaylists(track)
-    }
+    // --- FollowClient ---
+    override suspend fun isFollowing(item: EchoMediaItem): Boolean = false
+    override suspend fun getFollowersCount(item: EchoMediaItem): Long? = null
+    override suspend fun followItem(item: EchoMediaItem, shouldFollow: Boolean) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)